### ISSUE TITLE
Can't find module

### ISSUE THREAD

**Issue by mzch** [coralproject/talk#4226]
**Expected behavior:**
Coral talk starts successfully.

**Actual behavior:**
```
coral@ls:~/talk$ npm start

> @coralproject/talk@8.0.3 start
> NODE_ENV=production node dist/index.js

Error: Cannot find module 'core/client/framework/schema/__generated__/types'
Require stack:
- /srv/coral/talk/dist/core/server/data/cache/dataCache.js
- /srv/coral/talk/dist/core/server/graph/context.js
- /srv/coral/talk/dist/core/server/app/middleware/graphql/apolloServer.js
- /srv/coral/talk/dist/core/server/app/middleware/graphql/index.js
- /srv/coral/talk/dist/core/server/app/middleware/index.js
- /srv/coral/talk/dist/core/server/app/router/api/index.js
- /srv/coral/talk/dist/core/server/app/router/index.js
- /srv/coral/talk/dist/core/server/app/index.js
- /srv/coral/talk/dist/core/server/index.js
- /srv/coral/talk/dist/core/index.js
- /srv/coral/talk/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/srv/coral/talk/src/core/server/data/cache/dataCache.ts:7:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/srv/coral/talk/src/core/server/graph/context.ts:6:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
```

**Related Issues:** 

**Versions:**

- NodeJS: `v14.21.1`
- NPM: `8.19.3`
- MongoDB: `6.0.5`
- Redis: `7.0.10`

**Comment by nick-funk** [comment]
Hi @mzch, the reason you're seeing this is you're using npm version `8.19.3` if you check our `package.json` under `engines` it specifies you must use version `8.0.0` of `npm`. This is because our dependency installation is very specific and requires a work-around only available in npm `8.0.0` at this time (we are currently working to upgrade to node 18 + new npm, but it is not available right now).

**Comment by mzch** [comment]
I see. Thanks, @nick-funk !
